docs/content/docs/concepts/signals.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Signals and Signaler
 menuPosition: 6
-draft: true
```

Comment:
Woo! Excited to see this light up!

Reply:
Not sure why it was ever moved to draft. It used to be published.